fontir/src/orchestration.rs

```diff
@@ -116,6 +155,7 @@ impl Context {
 
     pub fn set_glyph_ir(&self, glyph_order: u32, ir: ir::Glyph) {
```

Comment:
question: why the WorkIdentifier::GlyphIr variant holds a u32 for the glyph id and not a String for the ir::Glyph.name instead?
If the purpose is that it be unique, the glyph name is by definition as unique as it can be and identifies the given ir::Glyph more than the glyph index.

Reply:
Idea was that WorkId is Copy and u32's are small. We could certainly have the name instead.

Reply:
Hm. In quickly trying to use name I found I had to copy it quite a lot so my inclination is to start with gid. I renamed the parameter.